test/e2e/mco_test.go

```diff
@@ -27,3 +31,63 @@ func TestClusterOperatorRelatedObjects(t *testing.T) {
 		t.Error("ClusterOperator.RelatedObjects should contain the MCO namespace")
 	}
 }
+
+func TestMastersSchedulable(t *testing.T) {
+	cs := framework.NewClientSet("")
+	schedulerCR, err := cs.ConfigV1Interface.Schedulers().Get("cluster", metav1.GetOptions{})
+	if err != nil {
+		t.Fatal("Error while listing scheduler CR")
+	}
+	schedulerCR.Spec.MastersSchedulable = true
+	if _, err = cs.ConfigV1Interface.Schedulers().Update(schedulerCR); err != nil {
+		t.Fatalf("Error while updating scheduler CR with error %v", err)
+	}
+	err = waitForAllMastersUpdate(cs, t, true)
+	if err != nil {
+		t.Fatalf("Expected all master nodes to be schedulable but it's not the case with %v", err)
+	}
+	// Reset scheduler CR
+	schedulerCR, err = cs.ConfigV1Interface.Schedulers().Get("cluster", metav1.GetOptions{})
+	if err != nil {
+		t.Fatal("Error while listing scheduler CR")
```

Comment:
minor: do we need the error added to the output here as well?

Reply:
Yeah, we should. It'll be useful later